R/tm_variable_browser.R

```diff
@@ -28,34 +28,55 @@
 #'   variable_browser_srv,
 #'   variable_browser
 #'
+#' @examples
+#' library(teal.widgets)
 #'
-#' @export
+#' # module specification used in apps below
+#' tm_variable_browser_module <- tm_variable_browser(
+#'   label = "Variable browser",
+#'   ggplot2_args = ggplot2_args(
+#'     labs = list(subtitle = "Plot generated by Variable Browser Module")
+#'   )
+#' )
 #'
-#' @examples
+#' # general data example
+#' data <- teal_data()
+#' data <- within(data, {
+#'   iris <- iris
+#'   mtcars <- mtcars
+#'   women <- women
+#'   faithful <- faithful
+#'   CO2 <- CO2
+#' })
+#' datanames(data) <- c("iris", "mtcars", "women", "faithful", "CO2")
+#'
+#' app <- init(
+#'   data = data,
+#'   modules = modules(tm_variable_browser_module)
+#' )
+#' if (interactive()) {
```

Comment:
there is error while running this example.
```
Listening on http://127.0.0.1:5675
[INFO] 2024-02-22 19:04:22.0670 pid:2104 token:[0b281f3f] teal Initializing reporter_previewer_module
Warning: Error in super$initialize: Assertion on 'label' failed: Must be of type 'character' (or 'NULL'), not 'list'.
  3: runApp
  2: print.shiny.appobj
  1: <Anonymous>
```

Reply:
https://github.com/insightsengineering/teal.modules.general/pull/651#issuecomment-1956677043

Reply:
🤦‍♂️ my bad

Reply:
: p